requirements/docs.txt

```diff
@@ -41,13 +41,13 @@ requests==2.18.4 \
     --hash=sha256:6a1b267aa90cac58ac3a765d067950e7dbbf75b1da07e895d1f594193a40a38b \
     --hash=sha256:9c443e7324ba5b85070c4a818ade28bfabedf16ea10206da1132edaa6dda237e
 # setuptools is required by Sphinx
-setuptools==36.4.0 \
-    --hash=sha256:4d54c0bfee283e78609169213f9c075827d5837086f58b588b417b093c23464b \
-    --hash=sha256:2758b0270fe8ceec42f336ee5b411e60dc8579febc27bb3ba9b794dc7f0239ae
+setuptools==36.5.0 \
+    --hash=sha256:ef824aefbd20dc364891836b75a19386dcf2f4235bf7d80531a8517ab29d0602 \
+    --hash=sha256:ce2007c1cea3359870b80657d634253a0765b0c7dc5a988d77ba803fc86f2c64
```

Comment:
can you rebase to master - all of the requirements changes should already be there

Reply:
True. I need to use a certain version of certifi though, as the requirements installed for the selenium tests need a newer version.